lib/services/UserService.js

```diff
@@ -0,0 +1,40 @@
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+const User = require('../models/Users');
+
+module.exports = class UserService {
+  static async create({ email, password }) {
+    const passwordHash = await bcrypt.hash(
+      password,
+      Number(process.env.SALT_ROUNDS)
+    );
+
+    const user = await User.insert({
+      email,
+      passwordHash,
+    });
+
+    return user;
+  }
+
+  static async signIn({ email, password }) {
+    try {
+      const user = await User.getByEmail(email);
+
+      if (!user) throw new Error('Invalid email');
+
+      const compare = bcrypt.compareSync(password, user.passwordHash);
+      if (compare === false) {
```

Comment:
you can just say ```if (compare)``` since if compare is ```false```, it will not run the body of the if statement